### PR TITLE
Update README and add appveyor.yml

### LINKED ISSUE #158: Create a Nano image
We need a Nano based image for aspnetcore and probably aspnetcore-build as well.

### PR DESCRIPTION
**Pull request by natemcmaster**
Resolves #158 
Resolves https://github.com/aspnet/aspnet-docker/issues/202

### CODE REVIEW

**Comment by MichaelSimons** [comment]
There is a corresponding nanoserver variant for each debian variant.  For `latest`, it is just `nanoserver`.  For each nanoserver tag, there is a corresponding tag that includes the nanoserver version.  Per Rich's direction, the nanoserver versioned variant is not included in the readme of microsoft/dotnet.

Here is the full list of tags I setup the build to produce:
1.0.4-nanoserver, 1.0.4-nanoserver-$(nanoserver.version), 1.0-nanoserver, 1.0-nanoserver-$(nanoserver.version), lts-nanoserver, lts-nanoserver-$(nanoserver.version)
1.1.1-nanoserver, 1.1.1-nanoserver-$(nanoserver.version), 1.1-nanoserver, 1.1-nanoserver-$(nanoserver.version), 1-nanoserver, 1-nanoserver-$(nanoserver.version), nanoserver, nanoserver-$(nanoserver.version)
1.0-1.1-2017-03-nanoserver, 1.0-1.1-2017-03-nanoserver-$(nanoserver.version), 1.0-1.1-nanoserver, 1.0-1.1-nanoserver-$(nanoserver.version)

File: 1.1/jessie/runtime/README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) image as its base.
 
-[![Downloads from Docker Hub](https://img.shields.io/docker/pulls/microsoft/aspnetcore.svg?style=flat-square)](https://hub.docker.com/r/microsoft/aspnetcore)
-[![Stars on Docker Hub](https://img.shields.io/docker/stars/microsoft/aspnetcore.svg?style=flat-square)](https://hub.docker.com/r/microsoft/aspnetcore)
+These images contain the runtime only. Use [`microsoft/aspnetcore-build`](https://hub.docker.com/r/microsoft/aspnetcore-build/) to build ASP.NET Core apps inside the container.
 
 ## Supported tags
 
 - [`1.1.1`, `1.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/runtime/Dockerfile)
+- [`1.1.1-nanoserver`, `1.1-nanoserver`, `1.1.1-nanoserver-10.0.14393.953` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/nanoserver/runtime/Dockerfile)
```
Comment (MichaelSimons) [inline 1.1/jessie/runtime/README.md]: nanoserver `latest` variant is not included.

File: 1.1/jessie/runtime/README.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 - [`1.0.4`, `1.0`, `lts` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/runtime/Dockerfile)
+- [`1.0.4-nanoserver`, `1.0-nanoserver`, `1.0.4-nanoserver-10.0.14393.953` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/nanoserver/runtime/Dockerfile)
```
Comment (MichaelSimons) [inline 1.1/jessie/runtime/README.md]: nanoserver `lts` variant is not included.

File: 1.1/jessie/runtime/README.md (hunk 1 above)
Comment (MichaelSimons) [inline 1.1/jessie/runtime/README.md]: Comment also applies to sdk readme

File: 1.1/jessie/runtime/README.md (hunk 2 above)
Comment (MichaelSimons) [inline 1.1/jessie/runtime/README.md]: Comment also applies to sdk readme

File: 1.1/jessie/sdk/README.md [hunk 3]
```diff
@@ -4,14 +4,14 @@ ASP.NET Core Build Docker Image
...
-
 ## Supported tags
 
 - [`1.1.1`, `1.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/sdk/Dockerfile)
+- [`1.1.1-nanoserver`, `1.1-nanoserver`, `1.1.1-nanoserver-10.0.14393.953` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/nanoserver/sdk/Dockerfile)
 - [`1.0.4`, `1.0`, `lts` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/sdk/Dockerfile)
-- [`1.0-1.1-2017-03`, `1.0-1.1` (designed for CI builds), (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/kitchensink/Dockerfile)
+- [`1.0.4-nanoserver`, `1.0-nanoserver`, `1.0.4-nanoserver-10.0.14393.953` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/nanoserver/sdk/Dockerfile)
+- [`1.0-1.1`, `1.0-1.1-2017-03` (designed for CI builds), (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/kitchensink/Dockerfile)
+- [`1.0-1.1-nanoserver`, `1.0-1.1-nanoserver-10.0.14393.953`(designed for CI builds), (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/nanoserver/kitchensink/Dockerfile)
```
Comment (MichaelSimons) [inline 1.1/jessie/sdk/README.md]: nanoserver `1.0-1.1-2017-03` variant is not included.

**Comment by MichaelSimons** [comment]
I guess I should remove the `1` variants I added to the build since they are not built for Debian.  That was an oversight on my part.

**Comment by natemcmaster** [comment]
We can add the `1` variant to debian.

**Comment by natemcmaster** [comment]
Updated

**Pull request merged**